liiatools_pipeline/sensors/location_schedule.py

```diff
@@ -54,6 +54,9 @@ def concat_directory_walker(folder_location, context, dataset):
             f"Failed to open folder location: {folder_location}/concatenated/{dataset}"
         )
         dir_contents = None
+    except Exception as err:
+        context.log.info(f"Some other error occurred {err}")
+        dir_contents = None
```

Comment:
If we return `None` for `dir_contents `then it will raise an error as we go on to loop through `dir_contents`. We can return an empty list but then we will get an error further down the line as it still can't find that folder. I wonder if it's best to add some code around line 131:

`directory_contents = input_directory_walker(folder_location, context, dataset)`

`if directory_contents is not None:`
    `for la_path, files in directory_contents.items():`

Reply:
I was mainly trying to work out what was causing the issue in staging.  I can remove this as the exception is actually too broad.  

In the logs I get: `Fixing episodes table failed: Cannot compare NaT with datetime.date object` so that's the error I'd rather have caught and managed than my code above.

Reply:
Up to you if you want to remove, although the above code (line 56) would still need updating. I know that wasn't from the PR, it looks like I made that change and didn't realise we'd run into an error from it

Reply:
Yep happy to make some changes here